### ISSUE TITLE
Temoa deploys too much capacity of certain technologies

### ISSUE THREAD

**Issue by robfairh** [arfc/pride#161]
PR #156 unveiled an issue. `Nbine` is a dummy technology with no associated cost. As it has no cost, it is free and Temoa deploys too much capacity of it. The following figures highlight this issue.

Figure 1 and 2 show the capacity and the generation respectively.
While the capacity is around 400 MW, the generation is only 200 GWh, meaning that an `Nbine` of only 23 MW would suffice.

Figure 1:
![capacity](https://user-images.githubusercontent.com/43791150/109293222-41919c80-780a-11eb-9549-6cd44cf64485.png)

Figure 2:
![generation](https://user-images.githubusercontent.com/43791150/109293227-43f3f680-780a-11eb-8a40-dbed2f5c6036.png)

This issue can be closed when modifications to the model are introduced to mitigate/solve this issue.

**Comment by robfairh** [comment]
Current behavior:
Figure 1 displays the electrical capacity. In this figure, the electrical capacity of `NUCLEAR` is represented by `NBINE`.
Figure 2 displays the steam capacity, where `NUCLEAR` represents the capacity to produce steam regardless of its use (whether it becomes USTM or UELC).

Figure 1: Electrical capacity
![costScenario_elc_capacity](https://user-images.githubusercontent.com/43791150/109305018-e5d00f00-781b-11eb-9b00-5f82e216ae75.png)

Figure 2: Industrial capacity
![costScenario_ind_capacity](https://user-images.githubusercontent.com/43791150/109305157-131cbd00-781c-11eb-9aea-fc4fb803153a.png)

Is this the desirable behavior? For the steam capacity, we are displaying all the available capacity of `NUCLEAR` and `ABBOTT`, while for the electrical capacity we are displaying only the capacity of producing electricity.
I believe we should be consistent and implement one of the following two options:
- Make the electrical capacity plot consider all the available capacity of the plant to produce electricity - i.e. the capacity to produce steam divided by the efficiency.
- Make the steam capacity plot consider only the available capacity of the plants that fulfill the `USTM` demand.

I believe the first approach is the easiest.
@samgdotson thoughts?

**Comment by robfairh** [comment]
I don't think what we've discussed yesterday IRL is going to work.
Currently, this plot shows what percentage of the `NSTM` goes into which technology:

![costScenario_reactor_generation](https://user-images.githubusercontent.com/43791150/109328892-b8458e80-7838-11eb-9c97-788be41d43c7.png)

If we used `TechOutputSplit` we would be constraining this coefficient to a constant value. According to these results, it is not constant throughout the simulation.
